sormas-api/src/main/java/de/symeda/sormas/api/importexport/ExportConfigurationDto.java

```diff
@@ -15,6 +15,7 @@
 	public static final String NAME = "name";
 
 	private String name;
+	private boolean reportsToPublic;
```

Comment:
why `reportsToPublic`? I don't see any meaning in context of export configuration. `shared` seems to be a better name.